app/_data/extensions/kong-inc/acme/versions.yml

```
@@ -0,0 +1 @@
+- release: 0.2.0
```

may need a newline if build fails

```suggestion
- release: 0.2.0

```